src/common/services/string.js

```diff
@@ -44,10 +44,12 @@ export const toSanitized = (str) => {
   if (!str) {
     return '';
   }
+  str = str.replace(/-/g, '');
   // replace all special characters | symbols with a space
   str = str.replace(/[`~!@#$%^&*()_\-+=[\]{};:'"\\|/,.<>?\s]/g, ' ');
```

Comment:
Why can't we make `-` as part of this regex batch itself? @koustov

Reply:
that one is about converting to `space`, here we are removing hyphen